src/lib/Footer.svelte

```diff
@@ -1,6 +1,51 @@
+<script lang="ts">
+  //optional link to a page about the site/service
+  export let aboutHref: string | undefined;
+  //optional link to privacy statement
+  export let privacyHref: string | undefined;
+  //optional link to accessiblity statement
+  export let accessibilityHref: string | undefined;
+  //optional link to cookie statement
+  export let cookiesHref: string | undefined;
+  $: aLinkExists =
```

Comment:
Style nit: just truthiness should be fine for undefined and strings, `x || y | | z`